<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="BOOK-KGT1HJ2FI7" failures="1" tests="5" name="com.qa.swaglabs.tests.LoginTest" time="10.050" errors="0" timestamp="2024-08-08T20:23:58 IST" skipped="2">
  <testcase classname="com.qa.swaglabs.tests.LoginTest" name="loginPageTitleTest" time="1.353">
    <skipped/>
  </testcase> <!-- loginPageTitleTest -->
  <system-out/>
  <testcase classname="com.qa.swaglabs.tests.LoginTest" name="loginPageTitleTest" time="1.459">
    <skipped/>
  </testcase> <!-- loginPageTitleTest -->
  <system-out/>
  <testcase classname="com.qa.swaglabs.tests.LoginTest" name="loginPageTitleTest" time="3.380">
    <failure message="expected [Swag Labs12] but found [Swag Labs]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [Swag Labs12] but found [Swag Labs]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at com.qa.swaglabs.tests.LoginTest.loginPageTitleTest(LoginTest.java:16)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:213)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1570)
]]>
    </failure>
  </testcase> <!-- loginPageTitleTest -->
  <system-out/>
  <testcase classname="com.qa.swaglabs.tests.LoginTest" name="loginIntoAppTest" time="1.927"/>
  <system-out/>
  <testcase classname="com.qa.swaglabs.tests.LoginTest" name="lockedOutUserTest" time="1.931"/>
  <system-out/>
</testsuite> <!-- com.qa.swaglabs.tests.LoginTest -->
